SYCL/Basic/regression/commandlist/gpu.cpp

```diff
@@ -0,0 +1,5 @@
+// REQUIRES: L0_plugin
```

Comment:
there test will not work as we do not have L0_plagin feature defined in lit.cfg.py

Reply:
cruft.  sorry about that.  i removed it.